src/parser/parser.yy

```diff
@@ -1951,11 +1957,11 @@ match_edge_type_list
     ;
 
 match_return
-    : KW_RETURN match_return_items match_order_by match_skip match_limit {
-        $$ = new MatchReturn($2, $3, $4, $5);
+    : KW_RETURN match_return_items match_sampling match_order_by match_skip match_limit {
+        $$ = new MatchReturn($2, $3, $4, $5, $6);
     }
-    | KW_RETURN KW_DISTINCT match_return_items match_order_by match_skip match_limit {
-        $$ = new MatchReturn($3, $4, $5, $6, true);
+    | KW_RETURN KW_DISTINCT match_return_items match_sampling match_order_by match_skip match_limit {
+        $$ = new MatchReturn($3, $4, $5, $6, $7, true);
```

Comment:
Could you give some test examples about the sampling feature in match statement?

Reply:
Here's an example of what I often use

> Go from 1 over e0 yield id($$) as id, toFloat(properties(edge).weight) as weight | limit 100 | sampling $-.weight 2
+-----+--------+
| id  | weight |
+-----+--------+
| 654 | 1.0    |
| 1   | 1.0    |
+-----+--------+
Got 2 rows (time spent 1514/2528 us)

I get the neighbors of the node with ID 1, and the weights of the connecting edges, and sample the neighbors through the weights, and finally get two values.